### PR TITLE
add limit option to list_folder

### PR DESCRIPTION
**Pull request by brianhenryhf**
adds limit option to list_folder (https://www.dropbox.com/developers/documentation/http/documentation#files-list_folder)

### CODE REVIEW

**Comment by Jesus** [comment]
That's looking good, thanks for your contribution.

I'll try to merge it over the weekend.

File: lib/dropbox_api/endpoints/files/list_folder.rb [hunk 1]
```diff
@@ -21,19 +21,25 @@ class ListFolder < DropboxApi::Endpoints::Rpc
...
+        :include_has_explicit_shared_members,
+        :limit
       ], options)
       options[:recursive] ||= false
       options[:include_media_info] ||= false
       options[:include_deleted] ||= false
       options[:shared_link] = build_shared_link_param(options[:shared_link]) if options[:shared_link]
-
+      options[:limit] = options[:limit] if options[:limit]
+
```
Comment (Jesus) [inline lib/dropbox_api/endpoints/files/list_folder.rb]: What does this line do?

Comment (brianhenryhf) [reply lib/dropbox_api/endpoints/files/list_folder.rb]: oops - not much.  that should be removed.  sorry about that.  I can PR with a removal, if you like.

Comment (Jesus) [reply lib/dropbox_api/endpoints/files/list_folder.rb]: Nevermind, I'll remove it when I get a chance.

Thanks for your contribution!

**Pull request merged**